docs/contribute/general/git.rst

```diff
@@ -114,37 +110,20 @@ Summary
 
    <prefix_name>: Summary
 
-The Subject shall describe what was changed in a single line max 72
-characters long. It shall include a prefix for the module, component or
-feature which was changed e.g. "doc:" or "bazel:" It shall start with a
-capital letter and should not be ended by a trailing period in the
-subject line.
+The Subject shall describe what was changed in a single line max 72 characters long. It shall not
+start or end with whitespace. It shall not end with a period.
 
 Good and bad examples for a subject are:
 
--  **mw: Show colorful output** not Add file
--  **bazel: Test Requirement SWS_CM_00001** not Add test
--  **osal: Split responsibilities of job handling and execution** not Refactor code
+-  **Show colorful output** not Add file
+-  **Test Requirement SWS_CM_00001** not Add test
+-  **Split responsibilities of job handling and execution** not Refactor code
 
 Description
 ===========
 
-The description must contain a brief summary of the content of the
-commit and why this is necessary. Furthermore it must be consistent and
-logically complete.
-
-If feasible, the commit message body should be extended with quoted
-material such as compiler warnings, debugger stack traces or measurement
-data for performance optimizations.
-
-The description may mention issues and link to them. A detailed description
-of linking commits to issues is available on `GitHub
-<https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue>`__.
-Be aware that keywords like (close | fix | resolve) will also close the
-referenced issue if the pull request is merged.
-
-Git commits are not required to mention issues. It is sufficient if the PR
-links to any relevant issues.
+The optional description can contain a brief summary of the content of the
+commit and why this is necessary.
```

Comment:
```suggestion
The optional description can be used to provide a brief summary of the content of the
commit and why this is necessary.
```

Rewording to make more clear that the description is optional, but if provided, is about a brief summary.